tests/make.zunit

```diff
@@ -20,3 +25,40 @@
     prefix=
     _fzf_complete_make 'make '
 }
+
+@test 'Testing output: make **' {
+    echo 0 > grep_mock_times
+
+    grep() {
+        grep_mock_times=$(($(cat grep_mock_times) + 1))
+        echo $grep_mock_times > grep_mock_times
+
+        grep_mock_$grep_mock_times $@
+    }
+
+    grep_mock_1() {
+        assert $# equals 3
+        assert $1 same_as '-E'
+        assert $2 same_as '^[a-zA-Z_-]+:.*?$$'
+        assert $3 same_as 'Makefile'
+
+        echo "grep_mock_$grep_mock_times error" >&2
+        return 1
+    }
+
+    _fzf_complete() {
+        assert $# equals 2
+        assert $1 matches '--ansi --tiebreak=index '
+        assert $2 same_as 'make '
+
+        run cat
+        assert ${#lines} equals 1
+    }
+
+    prefix=
+    _fzf_complete_make 'make ' 2> $tmpfile
+
+    run cat $tmpfile
+    assert ${#lines} equals 1
```

Comment:
This assertion is insufficient because empty lines are not properly counted. See how many hard I added empty lines to make this test fail that resulted in failure!

```zsh
_fzf_complete_make-target() {
    local fzf_options=$1
    shift

    echo >&2
    echo >&2
    echo >&2
    echo >&2
    echo >&2
    echo >&2
    echo >&2
    echo >&2
    echo >&2
    echo >&2
    _fzf_complete "--ansi --tiebreak=index $fzf_options" $@ < <(grep -E '^[a-zA-Z_-]+:.*?$$' Makefile 2> /dev/null | uniq | awk -F ':' '{ print $1 }'
}
```